custom_domains/getanid/variables.tf

```diff
@@ -0,0 +1,26 @@
+variable "zone" {}
+variable "front_door_name" {}
+variable "resource_group_name" {}
+variable "domains" {}
+variable "environment_tag" {}
+variable "ruleset_name" {}
+variable "environment_short" {}
+variable "service_name" {}
+variable "app_name" {}
+variable "app_resource_group_name" {}
+variable "transverse_sub_id" {}
+
```

Comment:
As discussed, can we try to reduce the number of variables and push some logic in the terraform module instead? This would simplify the code in this repo